kuma/wiki/jinja2/wiki/edit.html

```diff
@@ -156,7 +156,7 @@
 
         </header>
 
-        {% if request.user.wiki_revisions().count() == 0 and not revision_form.errors %}
+        {% if not revision_form.errors and not request.user.created_revisions.exists() %}
```

Comment:
I think the intent (if there was one) of using ``user.wiki_revisions()`` was to use the Django query cache to use the previous value.  Because there is an index on the ``user`` foreign key, ``user.created_revisions.exists()`` should be very fast, so there's no need to reuse the complex ``wiki_revisions`` query.